source/Octopus.Tentacle/Kubernetes/KubernetesConfig.cs

```diff
@@ -1,9 +1,11 @@
 ﻿using System;
+using Octopus.Tentacle.Util;
 
 namespace Octopus.Tentacle.Kubernetes
 {
     public static class KubernetesConfig
     {
+        public static bool IsRunningInKubernetesCluster => !Environment.GetEnvironmentVariable("KUBERNETES_SERVICE_HOST").IsNullOrEmpty();
```

Comment:
We have something like this already in the `PlatformDetection`

Reply:
Ah I didn't realise, thanks 👍